source/renderer/app/containers/profile/dialogs/BugReportDialogContainer.js

```diff
@@ -61,8 +65,8 @@ export default class BugReportDialogContainer extends Component<InjectedProps> {
         onGetLogs={() => {
           getLogs.trigger();
         }}
-        onCompressLogs={(logs) => {
-          compressLogs.trigger({ logs });
+        onGetFreshLogs={(logs) => {
```

Comment:
@daniloprates what is the difference between `onGetLogs` and `onGetFreshLogs`?
If the later one actually compresses the logs then I think the name should be changed...

Reply:
Renamed it to avoid confusion